healthyregions/SDOHPlace
# Add theme filters to the filter panel

This PR adds "Theme" filters to the filter panel, allowing users to directly explore different themes rather than only seeing them on the "No Results" page.

## How to Test
1. Go to the search page and open the filter panel. You should see a list of Theme filters:
<img width="839" alt="Screenshot 2024-11-05 at 3 07 33 PM" src="https://github.com/user-attachments/assets/55bebbcf-2337-4426-a2e4-bcae3572c5b4">

2. Click on any theme icon. The results list should update to show only items related to that theme. Note that all other filter and search settings will be reset:
<img width="942" alt="Screenshot 2024-11-05 at 3 07 59 PM" src="https://github.com/user-attachments/assets/82c1c070-4fb4-4941-ab62-ba2de591478c">
